docs/web3.contract.rst

```diff
@@ -952,6 +952,19 @@ For example:
 
 :py:class:`ContractEvent` provides methods to interact with contract events. Positional and keyword arguments supplied to the contract event subclass will be used to find the contract event by signature.
 
+.. _contract_get_logs:
+
+.. py:method:: ContractEvents.myEvent(*args, **kwargs).get_logs(filter_params, errors=WARN)
+   :noindex:
+
+   Fetches all logs for a given event within a set of filter params.
```

Comment:
might be worth linking to what valid `filter_params` look like here? We link out to many different places when referencing filter_params, so maybe worth standardizing too? Looks like we link out to either:
- [the execution api repo](https://github.com/ethereum/web3.py/pull/2968/files#diff-425d77e64ba190beb0e0b8066b0201e5fa798f148fb9d673e4f4e0827228e9f1R68)
- [the eth.filter documenation](https://web3py.readthedocs.io/en/stable/web3.eth.html#web3.eth.Eth.filter)
- or [the eth.org documentation](https://github.com/ethereum/web3.py/pull/2968/files#diff-425d77e64ba190beb0e0b8066b0201e5fa798f148fb9d673e4f4e0827228e9f1R203)

Reply:
good shout; i think potentially worthy of a dedicated section in our docs to dig into it and provide more interesting examples, but for now, i added a link to eth.org as an easy context add.